src/main/java/org/broadinstitute/hellbender/utils/IndexUtils.java

```diff
@@ -45,8 +47,10 @@ public static Index loadTribbleIndex(final File featureFile) {
     /**
      * Try to load the tabix index from disk, checking for out of date indexes and old versions
      * @return an Index, or null if we're unable to load
+     *
+     * @param errorOnOutOfDateIndex If true, will raise a UserException when an out of date index file is detected.
      */
-    public static Index loadTabixIndex(final File featureFile) {
+    public static Index loadTabixIndex(final File featureFile, final boolean errorOnOutOfDateIndex) {
```

Comment:
I realize its mostly in the test code, this is probably not a particularly important change